src/server/plugins/engine/pageControllers/PageControllerBase.ts

```diff
@@ -169,6 +167,11 @@ export class PageControllerBase {
     }
   }
 
+  get href() {
+    const { model, path } = this
+    return `/${model.basePath}${path}`
```

Comment:
Is there ever a case where we might have malformed URLs?

Reply:
Possibly as we support form definitions with `startPage` starting `http(s)://`

Not due to this PR, but we may have potential redirect issues lurking somewhere